src/Validators/DebugExportConfiguration.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Validators;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class DebugExportConfiguration
+{
+    /**
+     * @Assert\NotBlank
+     * @Assert\Regex(
+     *     pattern="/^[A-F0-9]{32}$/",
+     *     message="Invalid key provided."
+     * )
+     */
+    private $shopkey;
+
+    /**
+     * @Assert\NotBlank
+     * @Assert\Uuid(
```

Comment:
Cool, but is this really validating anything?

Reply:
Yes!
And with `strict=false`, it allows UUIDs without dashes in between.